### DISCUSSION TITLE
Not sure I understood layer handling

### DISCUSSION THREAD

**Discussion by conversy** [christofmuc/KnobKraft-orm#207] [General]
Hello,

I'm not sure I understood how to deal with layers. When I monitor which layer number is queried, I only get 0 or 1, even if `numberOfLayers` returns 4.
Shouldn't 2 or 3 be queried as well?

best.

**Comment by christofmuc** [comment]
Hm, yes it should, depending on where you look. I quickly tried an increased the number of layers for the DSI Prophet 12 from 2 to 4, and this is what I get:

![grafik](https://user-images.githubusercontent.com/5006524/213022053-bebb92dc-c574-409a-b101-52884deddaf5.png)

I think the main grid will not put more than 2 on a button, but the individual patch should allow to see and edit all 4?

**Comment by christofmuc** [comment]
Closing as answered.
